src/devices/tuya.ts

```diff
@@ -697,6 +697,43 @@ const definitions: Definition[] = [
             return exps;
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_dhke3p9w']),
+        model: 'F00YK04-18-1',
+        vendor: 'FORIA',
+        description: '18 scenes remote',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [e.battery(), e.battery_voltage(), e.action(['scene_1', 'scene_2', 'scene_3', 'scene_4', 'scene_5', 'scene_6',
```

Comment:
Does the battery and battery voltage work? I don't see it in `tuyaDatapoints`.

Reply:
The battery and voltage are not working yet

Reply:
Then please remove them for now.

Reply:
It's done

Reply:
It's done